pgex/examples/simple_projectile/_types.py

```diff
@@ -0,0 +1,13 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import typing as t
+
+import pygame
+
+PositionLike: t.TypeAlias = t.Union[
```

Comment:
What do you mean by `PositionLike` / `VelocityLike`? Why not only `Position`/`Velocity` that are more conventional?